integrations/github/src/definitions/events.ts

```diff
@@ -35,7 +36,26 @@ export const pullRequestMerged = {
   ui: {},
 }
 
+export const issueOpened = {
+  title: 'Issue opened',
+  description: 'Triggered when an issue is opened',
+  schema: z
+    .object({
+      type: z.literal(`${INTEGRATION_NAME}:issueOpened`).optional(),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏ remove type from the payload content